docs/standard/class-library-overview.md

```diff
@@ -38,68 +38,69 @@ ms.assetid: 7e4c5921-955d-4b06-8709-101873acf157
-||<xref:System.String>|An immutable, fixed-length string of Unicode characters.|**String**|**string**|**String^**|**string**|  
-  
- In addition to the base data types, the <xref:System> namespace contains over 100 classes, ranging from classes that handle exceptions to classes that deal with core runtime concepts, such as application domains and the garbage collector. The <xref:System> namespace also contains many second-level namespaces.  
-  
- For more information about namespaces, use the [.NET API Browser](../../api/index.md) to browse the .NET Class Library. The API reference documentation provides documentation on each namespace, its types, and each of their members.  
-  
+ The <xref:System> namespace is the root namespace for fundamental types in .NET. This namespace includes classes that represent the base data types used by all applications: <xref:System.Object> (the root of the inheritance hierarchy), <xref:System.Byte>, <xref:System.Char>, <xref:System.Array>, <xref:System.Int32>, <xref:System.String>, and so on. Many of these types correspond to the primitive data types that your programming language uses. When you write code using .NET types, you can use your language's corresponding keyword when a .NET base data type is expected.
+
+ The following table lists the base types that .NET supplies, briefly describes each type, and indicates the corresponding type in Visual Basic, C#, C++, and F#.
+
+|Category|Class name|Description|Visual Basic data type|C# data type|C++/CLI data type|F# data type|
+|--------------|----------------|-----------------|----------------------------|-------------------|---------------------|-----------------------|
+|Integer|<xref:System.Byte>|An 8-bit unsigned integer.|**Byte**|**byte**|**unsigned char**|**byte**|
+||<xref:System.SByte>|An 8-bit signed integer.<br /><br /> Not CLS-compliant.|**SByte**|**sbyte**|**char**<br /> -or-<br /> **signed** **char**|**sbyte**|
+||<xref:System.Int16>|A 16-bit signed integer.|**Short**|**short**|**short**|**int16**|
+||<xref:System.Int32>|A 32-bit signed integer.|**Integer**|**int**|**int**<br /><br /> -or-<br /><br /> **long**|**int**|
+||<xref:System.Int64>|A 64-bit signed integer.|**Long**|**long**|**__int64**|**int64**|
+||<xref:System.UInt16>|A 16-bit unsigned integer.<br /><br /> Not CLS-compliant.|**UShort**|**ushort**|**unsigned short**|**uint16**|
+||<xref:System.UInt32>|A 32-bit unsigned integer.<br /><br /> Not CLS-compliant.|**UInteger**|**uint**|**unsigned int**<br /> -or-<br /> **unsigned long**|**uint32**|
+||<xref:System.UInt64>|A 64-bit unsigned integer.<br /><br /> Not CLS-compliant.|**ULong**|**ulong**|**unsigned __int64**|**uint64**|
+|Floating point|<xref:System.Half>|A half-precision (16-bit) floating-point number.|||||
+||<xref:System.Single>|A single-precision (32-bit) floating-point number.|**Single**|**float**|**float**|**float32**<br> or<br>**single**|
```

Comment:
Should `Half` be listed here as well? It isn't currently a language primitive but it is a "fundamental type" in that it is part of the ABI for various platforms and may become a language primitive in the future.

The same would be true if we added `Int128` and `UInt128` types in the future, for example.

Reply:
It's there already on line 89.

Reply:
Completely missed it 😅